releasenotes/releasenotes_v5.rst

```diff
@@ -5,23 +5,91 @@ Current Firely Server release notes (v5.x)
 
 .. _vonk_releasenotes_5_1_0:
 
-Release 5.1.0, April xth, 2023
+Release 5.1.0, May xth, 2023
 ------------------------------
 
+Firely Server 5.1.0 brings support for Bulk Data Export 2.0, Electronic Health Information Export (b)(10) and several other features.
+
+Existing installations may be affected by the fixes on Composite search parameters for the SQL Server database repository.
+
+Database
+^^^^^^^^
+
+* The SQL Server database schema is upgraded from version 26 to 27. The upgrade will be applied automatically, but if you have a very large database you may want to apply it manually using the script FS_SchemaUpgrade_Data_v26_v27.
+* This implies that you also need to upgrade Firely Server Ingest to version 2.2.0, to match the new database schema.
+
+Configuration
+^^^^^^^^^^^^^
+
+* The ``HistoryOptions`` are removed as configuration option, so you can remove it from your configuration in ``appsettings.instance.json`` or environment variables as well. The returned resources will be limited by the settings in the ``BundleOptions``, see :ref:`bundle_options`.
+* The Bulk Data Export upgrades (see below) come with a few extra configuration settings, see :ref:`feature_bulkdataexport`
+
+Features
+^^^^^^^^
+* Firely Server is upgraded to the release version (5.0.0) of FHIR R5. If you have your administration database in SQL Server or MongoDB, this means that the conformance resources will be :ref:`re-imported <conformance_import>`.
+* We included ``errataR5.zip`` with fixes to a few resources and search parameters that have errors in the specification. These are imported automatically at startup.
+* We upgraded Firely Server to the latest SDK 5.1.0, see its `releasenotes <https://github.com/FirelyTeam/firely-net-sdk/releases/tag/v5.1.0>`_.
+* Support for §170.315(b)(10) - Electronic Health Information Export, as required by the end of 2023.
```

Comment:
I think we should only publish this sentence once we have the official CHPL listing